hedera-node/hedera-network-service-impl/src/main/java/com/hedera/node/app/service/network/impl/NetworkServiceImpl.java

```diff
@@ -19,29 +19,155 @@
+     */
+    public NetworkServiceImpl() {
+        this.networkGetAccountDetailsHandler = new NetworkGetAccountDetailsHandler();
+        this.networkGetByKeyHandler = new NetworkGetByKeyHandler();
+        this.networkGetExecutionTimeHandler = new NetworkGetExecutionTimeHandler();
+        this.networkGetVersionInfoHandler = new NetworkGetVersionInfoHandler();
+        this.networkTransactionGetReceiptHandler = new NetworkTransactionGetReceiptHandler();
+        this.networkTransactionGetRecordHandler = new NetworkTransactionGetRecordHandler();
+        this.networkUncheckedSubmitHandler = new NetworkUncheckedSubmitHandler();
+    }
+
+    /**
+     * Returns the {@link NetworkGetAccountDetailsHandler} instance.
+     *
+     * @return the {@link NetworkGetAccountDetailsHandler} instance.
+     */
+    @NonNull
+    public NetworkGetAccountDetailsHandler getNetworkGetAccountDetailsHandler() {
+        return networkGetAccountDetailsHandler;
+    }
+
+    /**
+     * Returns the {@link NetworkGetByKeyHandler} instance.
+     *
+     * @return the {@link NetworkGetByKeyHandler} instance.
+     */
+    @NonNull
+    public NetworkGetByKeyHandler getNetworkGetByKeyHandler() {
+        return networkGetByKeyHandler;
+    }
+
+    /**
+     * Returns the {@link NetworkGetExecutionTimeHandler} instance.
+     *
+     * @return the {@link NetworkGetExecutionTimeHandler} instance.
+     */
+    @NonNull
+    public NetworkGetExecutionTimeHandler getNetworkGetExecutionTimeHandler() {
+        return networkGetExecutionTimeHandler;
+    }
+
+    /**
+     * Returns the {@link NetworkGetVersionInfoHandler} instance.
+     *
+     * @return the {@link NetworkGetVersionInfoHandler} instance.
+     */
+    @NonNull
+    public NetworkGetVersionInfoHandler getNetworkGetVersionInfoHandler() {
+        return networkGetVersionInfoHandler;
+    }
+
+    /**
+     * Returns the {@link NetworkTransactionGetReceiptHandler} instance.
+     *
+     * @return the {@link NetworkTransactionGetReceiptHandler} instance.
+     */
+    @NonNull
+    public NetworkTransactionGetReceiptHandler getNetworkTransactionGetReceiptHandler() {
+        return networkTransactionGetReceiptHandler;
+    }
+
+    /**
+     * Returns the {@link NetworkTransactionGetRecordHandler} instance.
+     *
+     * @return the {@link NetworkTransactionGetRecordHandler} instance.
+     */
+    @NonNull
+    public NetworkTransactionGetRecordHandler getNetworkTransactionGetRecordHandler() {
+        return networkTransactionGetRecordHandler;
+    }
+
+    /**
+     * Returns the {@link NetworkUncheckedSubmitHandler} instance.
+     *
+     * @return the {@link NetworkUncheckedSubmitHandler} instance.
+     */
+    @NonNull
+    public NetworkUncheckedSubmitHandler getNetworkUncheckedSubmitHandler() {
+        return networkUncheckedSubmitHandler;
+    }
+
+    @NonNull
+    @Override
+    public Set<TransactionHandler> getTransactionHandler() {
+        return Set.of(networkUncheckedSubmitHandler);
```

Comment:
Is this the unchecked submit handler we discussed removing?

Reply:
No idea